### PR TITLE
Various Design Polish

### PR DESCRIPTION
**Pull request by jscheel**
This PR contains some various small items of design polish throughout the project.

Fix border prop on avatar

Add ability to disable hovering on compact avatar list

Remove visited link styles

Fade out bottom of median

Tighten padding on inputs

Darken gunGrey base color

Shorten animation durations

Fix gradient below median

Clean up loader

Add delay option to loader

### CODE REVIEW

File: src/components/Container.js [hunk 1]
```diff
@@ -52,15 +74,15 @@ const contentAnimationVariants = {
     transition: {
       when: 'beforeChildren',
       type: 'tween',
-      duration: 0.2,
+      duration: 0.15,
```
Comment (assimovt) [inline src/components/Container.js]: if durations are used consistently, shall we move them out to the base styles?

File: src/components/Container.js [hunk 2]
```diff
@@ -32,11 +33,32 @@ const StyledContent = styled(motion.div)`
...
+    border-top: 3px solid ${props => colors[props.gradientColor]};
+    background: linear-gradient(
+      ${props => colors[props.gradientColor]},
+      ${props =>
+        Color(colors[props.gradientColor])
+          .fade(1)
+          .rgb()
+          .string()}
+    );
+    z-index: 9999999;
```
Comment (assimovt) [inline src/components/Container.js]: Why would we need z-index here? The widget has an option to override a z-index, would this conflict then?

Comment (jscheel) [reply src/components/Container.js]: This is specifically for the median, which is the container for the search bar

File: src/components/Container.js (hunk 1 above)
Comment (jscheel) [reply src/components/Container.js]: They aren't entirely consistent, but I can unify some

**Pull request merged**